### PR TITLE
Check balance in rental period component

### PR DESCRIPTION
**Pull request by javiesses**

### CODE REVIEW

File: src/app/tabs/registrar/operations.js [hunk 1]
```diff
@@ -15,16 +15,27 @@ import { notifyError, notifyTx, txTypes } from '../../notifications';
...
+  const web3 = new Web3(window.ethereum);
+  const rif = new web3.eth.Contract(rifAbi, rifAddress);
+
+  const registrar = new web3.eth.Contract(fifsRegistrarAbi, fifsRegistrarAddress);
 
   dispatch(requestGetCost(duration));
 
   return new Promise((resolve) => {
-    registrar.price(domain, 0, duration, (error, result) => {
+    registrar.methods.price(domain, 0, duration).call((error, cost) => {
```
Comment (ilanolkies) [inline src/app/tabs/registrar/operations.js]: Unnecessary Promises. Chain the promises using `.then`.

File: src/config/contracts.json [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
-  "registrar": "0x5269f5bc51cdd8aa62755c97229b7eeddd8e69a6",
-  "reverseRegistrar": "0xd25c3f94a743b93ecffecbe691beea51c3c2d9d1",
-  "publicResolver": "0x4efd25e3d348f8f25a14fb7655fba6f72edfe93a",
-  "nameResolver": "0x4b1a11bf6723e60b9d2e02aa3ece34e24bde77d9",
-  "multiChainResolver": "0x99a12be4c89cbf6cfd11d1f2c029904a7b644368",
-  "rif": "0x2acc95758f8b5f583470ba265eb685a8f45fc9d5",
-  "fifsRegistrar": "0x779195c53cc7c1a33bd2eea5f63f2c1da8798d61",
-  "rskOwner": "0x45d3e4fb311982a06ba52359d44cb4f5980e0ef1",
-  "renewer": "0x779195c53cc7c1a33bd2eea5f63f2c1da8798d61",
+  "rns": "0x830C1a4aC017Aa9734674169B6738dc80E799A51",
```
Comment (ilanolkies) [inline src/config/contracts.json]: Wrong addresses.

File: src/config/nodes.json [hunk 3]
```diff
@@ -1,3 +1,3 @@
 {
-  "rskMain": "https://public-node.rsk.co"
+  "rskMain": "http://localhost:7545"
```
Comment (ilanolkies) [inline src/config/nodes.json]: Don’t change public node here

File: src/languages/en.json [hunk 4]
```diff
@@ -209,5 +209,8 @@
   "not_registered_domain": "The domain has not been registered or has expired, you should register it before renewing.",
   "start_renovation_for": "start renovation for",
   "renew": "renew",
-  "renew_domain_successfully": "the domain has been renewed succesfully"
+  "renew_domain_successfully": "the domain has been renewed succesfully",
+  "not_enough_balance": "you don't have enough balance to perform this action",
```
Comment (ilanolkies) [inline src/languages/en.json]: Merging when i18n

**Comment by ilanolkies** [comment]
Closes #68

**Comment by javiesses** [comment]
Close #68

**Pull request merged**